### ISSUE TITLE
`process.argv` returns `[native code]`

### ISSUE THREAD

**Issue by theseyan** [oven-sh/bun#1108]
### Version

0.1.10

### Platform

Linux 5.10.102.1-microsoft-standard-WSL2 #1 SMP Wed Mar 2 00:30:59 UTC 2022 x86_64 GNU/Linux

### What steps will reproduce the bug?

```bash
echo "console.log(require('process').argv)" > bug.js && bun bug.js
```

### How often does it reproduce? Is there a required condition?

It is reproducible everytime.
This bug is present only when `process` is `require`d or `import`ed. Global `process.argv` works fine and returns an array.

### What is the expected behavior?

`require('process').argv` returns an array instead of `[native code]`

### What do you see instead?

`require('process').argv` returns  `[native code]`

**Comment by DerGoogler** [comment]
Use 
```js
const { argv } = process
```

and to env's
```js
const { argv, env } = process
```

**Comment by theseyan** [comment]
Yeah, that works fine but the fact that `require('process').argv` returns wrong is a bug regardless

**Comment by Jarred-Sumner** [comment]
Fixed in bun v0.1.11
